dbt/include/trino/macros/materializations/incremental.sql

```diff
@@ -61,43 +130,43 @@
     )
 {%- endmacro %}
 
-{% macro trino__get_merge_sql(target, source, unique_key, dest_columns, predicates=none) -%}
+{% macro trino__get_merge_sql(target, source, unique_key, dest_columns, predicates) -%}
```

Comment:
do we have any differences here compared to the default dbt implementation?

i remember we had found a bug in a previous Trino implementation which caused to rewrite the macro in Trino

cc @hovaesco

Reply:
We had two issues:

* We would not use false as a merge match condition
* Ambiguous columns in merge match condition